lms/static/js/courseware/certificates_api.js

```diff
@@ -0,0 +1,18 @@
+$(document).ready(function() {
+    $("#btn_generate_cert").click(function(e){
+        var post_url = $("#btn_generate_cert").data("endpoint");
```

Comment:
It might be a good idea to `e.preventDefault();` here to stop event propagation.